Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -156,7 +156,6 @@ public class LiveSessionCoordinator<R: RootRegistry>: ObservableObject {
         internalState = .notConnected(reconnectAutomatically: false)
     }
     
-    @_spi(NarwinChat)
     public func reconnect() async {
```

Comment:
Can you add docs for this noting what it does and when it should generally be used?